style/ruby/README.md

```diff
@@ -38,6 +38,9 @@ Ruby
 * Use heredocs for multi-line strings.
 * Prefer `protected` over `private` for non-public `attr_reader`s, `attr_writer`s,
   and `attr_accessor`s.
+* Break up method definitions over 80 characters long by
```

Comment:
I've updated the PR to have the the breaking up argument list section to link to the two examples of it.